### DISCUSSION TITLE
postgresql_home_dir

### DISCUSSION THREAD

**Discussion by ayadahm** [autobase-tech/autobase#619] [General]
Changing the default home directory for postgres to other than /var/lib/postgresql in ubuntu won't work. is this known issue ?

**Comment by vitabaks** [comment]
when installing postgresql packages, a postgres user is automatically created with his home directory. If you want to specify a different home directory, you need to create a user first or explicitly change the home directory for an already created user. 

But I haven't tried to change it because I don't see the need for it.

**Reply by ayadahm** [reply]
Yes .
I changed the home directory afterwards exactly as you said. 

Many thanks 😊
